### PR TITLE
Move tutorial and getting started pages, small IA updates

### PR DESCRIPTION
**Pull request by dacharyc**
## Pull Request Info

- [MongoDB App Services Updates - Staged](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/get-started-tutorial-ia-updates/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/introduction.txt [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+Secure Client-Side Queries
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Developers need efficient, secure data access.
+If the data is on a server, you need an API to access the data and prevent unauthorized access.
+
+The Realm Web SDK includes a :ref:`secure MongoDB client <intro-web-mql>` to interact with data. It uses
+:ref:`GraphQL <intro-web-graphql>` and the :manual:`Query API </tutorial/query-documents>`
+to access data in {+atlas+}. Use either or both of these APIs to work with
+your application's data.
```
Comment (mongodben) [inline source/introduction.txt:143]: we could generalize it a bit more to be less web-centric, more general client-side talking about other SDKs

File: source/introduction.txt [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+- Call from client applications with the Realm Web SDK.
+- Execute Realm Functions in response to trigger events, like an update to a document in MongoDB or at a scheduled time.
+- HTTPS endpoints that execute a function in response to external applications.
+
+To learn more about defining and using serverless functions, see
+:ref:`functions` and :ref:`triggers`.
+
+.. _intro-web-query:
+
+Secure Client-Side Queries
```
Comment (mongodben) [inline source/introduction.txt:134]: there should be a subsection here about using realm sync as well if not a dedicated sync section, if we're also talking about MQL and GraphQL. can prob steal smthn abt this from the intro for mobile developers

File: source/introduction.txt [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+Get Started with MongoDB Realm
+------------------------------
+
+You can ship your first app built with MongoDB Realm in minutes.
+
+Tutorials
+~~~~~~~~~
+
+Check out our :doc:`tutorials </tutorials>` to learn more about how to build 
+Realm Applications. Tutorials start with git repositories you can clone
```
Comment (mongodben) [inline source/introduction.txt:219]:
```suggestion
Realm applications. Tutorials start with git repositories you can clone
```

File: source/introduction.txt [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+and run, and walk you through code excerpts that are specific to working
+with Realm.
+
+To get started with your first web app, follow the :ref:`Task Tracker 
+application tutorial <tutorial-task-tracker-web>`. You can also refer to the 
+:ref:`Web SDK <web-intro>` documentation for TypeScript and JavaScript.
```
Comment (mongodben) [inline source/introduction.txt:225]: could this be generalized for all Realm client SDKs?

File: source/introduction.txt [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+~~~~~~~~~~~~~~~~~~~~~
+
+To explore a prebuilt app that you can customize, create a template app.
+Template applications are working apps that use Realm services to do things 
+like:
+
+- Use Sync to synchronize data between mobile clients and the linked 
+  MongoDB Atlas collection.
+- Host a Todo web app using the :ref:`GraphQL API <graphql-api>`
+- Host a Todo web app using the :ref:`Realm Web SDK <web-intro>`
```
Comment (mongodben) [inline source/introduction.txt:240]:
```suggestion
- Host a Todo web app using the :ref:`Realm Web SDK <web-intro>`
- Create a Todo mobile application using the `Realm client SDKs <https://www.mongodb.com/docs/realm/>`__
```

curiously there's no ref for the Realm SDKs homepage, so providing URL link to it here. maybe there's a better way tho..

File: source/introduction.txt [hunk 6]
```diff
@@ -0,0 +1,263 @@
...
+  custom permissions via rules, and authentication via third-party SSO.
+- {+service+}'s built-in :ref:`authentication system <users-and-authentication>` 
+  simplifies user data access.
+  You can add users to your app and define :ref:`permissions
+  <permissions>` that control who can read and write data.
+- Backend use cases for {+service+} include:
+
+  - Crunching data
+  - Reacting to data changes in MongoDB Atlas
+  - Interacting with third-party services.
```
Comment (mongodben) [inline source/introduction.txt:263]:
```suggestion
  - Interacting with third-party services
```

File: source/introduction.txt (hunk 4 above)
Comment (dacharyc) [reply source/introduction.txt:225]: Now that the sites are separated, and with the plan to deprecate the existing task tracker tutorials, I didn't want to link to them here. Instead, I _just_ wanted to link to the stuff that's relevant to App Services + the template apps I linked to in the next section.

File: source/introduction.txt [hunk 7]
```diff
@@ -0,0 +1,263 @@
...
+application tutorial <tutorial-task-tracker-web>`. You can also refer to the 
+:ref:`Web SDK <web-intro>` documentation for TypeScript and JavaScript.
+
+To get started with a backend app, follow our :ref:`GitHub Issue Tracker
+tutorial <backend-tutorial>`. 
+
+Prebuilt Applications
+~~~~~~~~~~~~~~~~~~~~~
+
+To explore a prebuilt app that you can customize, create a template app.
```
[the remark this answers is not in the document]
Comment (dacharyc) [reply source/introduction.txt:233]: Ah, good call! I worked on the intro before I mucked with the template apps page and forgot to go back and add a link. Thanks for mentioning it!

File: source/introduction.txt (hunk 4 above)
Comment (mongodben) [reply source/introduction.txt:255]: it just reads weird to me to refer only to the web sdk w/o mention of the others here.

File: source/introduction.txt [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+Realm apps can react to changes in your `{+atlas+}
+<https://www.mongodb.com/realm?tck=docs_realm#atlas-form-container>`_ data, connect that data to other systems,
+and scale to meet demand. {+service-short+} does this without the need 
+to manage database and server infrastructure.
+
+As a web developer, you have to deal with lots of overhead when sending 
+data from a server to the browser. This includes things like authentication 
+and data validation. And then the application has to be scalable and secure. 
+Many libraries and frameworks exist to provide these services.
+Selecting the best ones for your project requires deep understanding of the tradeoffs.
```
[the remark this answers is not in the document]
Comment (mongodben) [reply source/introduction.txt:25]: then maybe add another small paragraph abt 'as a mobile developer' to cover all the bases here

**Pull request merged**